debian/additions/mariadb.conf.d/50-server.cnf

```diff
@@ -30,94 +29,80 @@ bind-address            = 127.0.0.1
 #
 # * Fine Tuning
 #
+
 #key_buffer_size        = 16M
-#max_allowed_packet     = 16M
+#max_allowed_packet     = 1G
 #thread_stack           = 192K
 #thread_cache_size      = 8
 # This replaces the startup script and checks MyISAM tables if needed
 # the first time they are touched
 #myisam_recover_options = BACKUP
 #max_connections        = 100
 #table_cache            = 64
-#thread_concurrency     = 10
-
-#
-# * Query Cache Configuration
-#
-#query_cache_limit      = 1M
-query_cache_size        = 16M
 
 #
 # * Logging and Replication
 #
+
 # Both location gets rotated by the cronjob.
 # Be aware that this log type is a performance killer.
-# As of 5.1 you can enable the log at runtime!
+# Recommend only changing this at runtime for short testing periods if needed!
 #general_log_file       = /var/log/mysql/mysql.log
 #general_log            = 1
-#
+
 # Error log - should be very few entries.
-#
 log_error = /var/log/mysql/error.log
-#
+
 # Enable the slow query log to see queries with especially long duration
 #slow_query_log_file    = /var/log/mysql/mariadb-slow.log
 #long_query_time        = 10
 #log_slow_rate_limit    = 1000
-#log_slow_verbosity     = query_plan
+#log_slow_verbosity     = query_plan,explain
 #log-queries-not-using-indexes
-#
+#min_examined_row_limit = 1000
+
 # The following can be used as easy to replay backup logs or for replication.
 # note: if you are setting up a replication slave, see README.Debian about
 #       other settings you may need to change.
 #server-id              = 1
 #log_bin                = /var/log/mysql/mysql-bin.log
 expire_logs_days        = 10
 #max_binlog_size        = 100M
-#binlog_do_db           = include_database_name
-#binlog_ignore_db       = exclude_database_name
+
 
 #
-# * Security Features
-#
-# Read the manual, too, if you want chroot!
-#chroot = /var/lib/mysql/
+# * SSL/TLS
 #
+
 # For generating SSL certificates you can use for example the GUI tool "tinyca".
-#
 #ssl-ca = /etc/mysql/cacert.pem
 #ssl-cert = /etc/mysql/server-cert.pem
 #ssl-key = /etc/mysql/server-key.pem
-#
+
 # Accept only connections using the latest and most secure TLS protocol version.
-# ..when MariaDB is compiled with OpenSSL:
-#ssl-cipher = TLSv1.2
-# ..when MariaDB is compiled with YaSSL (default in Debian):
+# ..when MariaDB is compiled with OpenSSL (default in mariadb.org packaging):
+#ssl-cipher = TLSv1.3
```

Comment:
https://mariadb.com/kb/en/ssltls-system-variables/#tls_version

I suspect openssl will allow the tls version in a cipher but wolfssl won't.

noting https://mariadb.com/kb/en/using-tlsv13/ might be approprate if you include ssl-cipher here still.

Reply:
I'll just remove the ssl-cipher section and refer to https://mariadb.com/kb/en/securing-connections-for-client-and-server/